### PR TITLE
repr added to DipoleFixed.

### PR DESCRIPTION
**Pull request by rodrigohubner**
Fix this issue: https://github.com/mne-tools/mne-python/issues/4149

### CODE REVIEW

**Comment by rodrigohubner** [comment]
@Eric89GXL @agramfort 
I didn't understand the error occurred in  continuous-integration/travis-ci/pr.
Can you help me?

File: mne/tests/test_dipole.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 from mne.surface import _compute_nearest
 from mne.bem import _bem_find_surface, read_bem_solution
 from mne.transforms import apply_trans, _get_trans
+from mne.beamformer import rap_music
```
Comment (agramfort) [inline mne/tests/test_dipole.py]: don't use rap_music in a test that is unrelated to rap_music. Just add a print statement in a test that has a DipoleFixed object.

Comment (rodrigohubner) [inline mne/tests/test_dipole.py]: I just fixed it. Thank you.

**Comment by rodrigohubner** [comment]
Ok. Is it already to accept?

**Comment by Eric89GXL** [comment]
Thanks @rodrigohubner

**Comment by jaeilepp** [comment]
FYI @rodrigohubner magic word 'closes' makes the related issue close automatically.
